pkg/plugins/resources/bitbucket/pullrequest/main.go

```diff
@@ -0,0 +1,112 @@
+package pullrequest
+
+import (
+	"github.com/mitchellh/mapstructure"
+	"github.com/updatecli/updatecli/pkg/plugins/resources/bitbucket/client"
+	giteascm "github.com/updatecli/updatecli/pkg/plugins/scms/bitbucket"
```

Comment:
```suggestion
	bitbucketscm "github.com/updatecli/updatecli/pkg/plugins/scms/bitbucket"
```

This suggestion must be applied in the file